src/fat/bios_param.rs

```diff
@@ -0,0 +1,30 @@
+
+#[derive(Clone,Debug)]
+#[repr(C)]
+pub struct BIOSParam {
+	pub bytes_per_sector: u16,
+	pub sectors_per_cluster: u8,
+	pub reserved_sectors: u16,
+	pub fat_count: u8,
+	pub max_roots: u16,
+	pub sectors: u16,
+	pub media_id: u8,
+	pub sectors_per_fat: u16,
+}
+
+/*
```

Comment:
Sure. Will complete.  I planned on adding some convenience calculation functions to the bios_param code but didn't get around to it before submitting the PR.  I'll add it in or remove commented code.